OpenAPI/kiota/install.md

```diff
@@ -11,12 +11,20 @@ date: 03/10/2023
 
 Kiota can be accessed in the following ways.
 
+**Microsoft provided options:**
+
 - [Download binaries](#download-binaries)
 - [Run in Docker](#run-in-docker)
 - [Install as .NET tool](#install-as-net-tool)
 - [Build from source](#build-from-source)
 - [Install the Visual Studio Code extension (preview)](#install-the-visual-studio-code-extension)
 
+**Community provided options:**
+
+> Disclaimer: The community provided options are maintained and distributed by the community and are not official Microsoft installation options. Microsoft makes no warranties, express or implied, with respect to the community installation options or their use. Use of these installation options is at your own risk. Microsoft shall not be liable for any damages arising out of or in connection with the use of these installation options.
```

Comment:
This text is very very defensive on the Microsoft side. Though, I'm wondering if all this caution is really necessary for "just" an installation mechanism ...
Saying that it's a community effort and not maintained, nor supported by Microsoft isn't enough to settle expectations without the risk of scary ppl?

Reply:
Stating that's community maintained already removes Microsoft from the equation imho.
In all honesty this wording looks pretty scary and I haven't seen this kind of defensiveness in other Microsoft projects, to the best of my knowledge.
E.g. this is a page in the TS documentation directly linking external projects: https://www.typescriptlang.org/docs/bootstrap

Anyhow, I'm not trying to block anything here, just noticing a much more strict wording than expected.

Reply:
this is something that I got from our internal legal resources. If we want something custom we probably need to involve the legal team, which is going to take weeks if not months.

Reply:
I understand, what do you think about going with this wording to start with, but internally starting along the process with legals?
I think that, providing evidence of other Microsoft projects using a more relaxed approach, we have a good chance to be able to improve here (hopefully not triggering scrutiny on other projects though!)

Reply:
It might if we use them as example. Alright let's roll with that, and I'll ask @RabebOthmani if she can start talking to the legal team in all of her spare time :)